docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
+        [page:.weight weight]=0, [page:.timeScale timeScale]=0) can prevent the animation from playing, 
+        too.
+    </div>
 
-    <h3>[method:AnimationAction crossFadeTo]( [page:AnimationAction fadeInAction], [page:Number duration], [page:Boolean warp] )</h3>
+    <h3>[method:AnimationAction reset]()</h3>
+    <div>
+        Resets the action. This method can be chained.<br /><br />
+        
+        This method sets [page:.paused paused] to false, [page:.enabled enabled] to true, 
+        [page:.time time] to 0, interrupts any scheduled fading and warping, and removes the internal 
+        loop count and scheduling for delayed starting.<br /><br />
+
+        Note: .*reset* is always called by [page:.stop stop], but .*reset* doesn’t call .*stop* itself.
+        This means: If you want both, resetting and stopping, don’t call .*reset*; call .*stop* instead.
+    </div>
 
-    <h3>[method:AnimationAction stopFading]()</h3>
+    <h3>[method:AnimationAction setDuration]( [page:Number durationInSeconds] )</h3>
+    <div>
+        Sets the duration for a single loop of this action (by adjusting [page:.timeScale timeScale] 
+        and stopping any scheduled warping). This method can be chained.
+    </div>
 
     <h3>[method:AnimationAction setEffectiveTimeScale]( [page:Number timeScale] )</h3>
+    <div>
+        Sets the [page:.timeScale timeScale] and stops any scheduled warping. This method can be chained.<br /><br />
+        
+        If [page:.paused paused] is false, the effective time scale (an internal property) will be set 
+        to this value, too; otherwise the effective time scale (directly affecting the animation at 
+        this moment) will be set to 0.<br /><br />
+        
+        Note: .*paused* will not be switched to *true* automatically, if .*timeScale* is set to 0 by 
+        this method.
+    </div>    
 
-    <h3>[method:Number getEffectiveTimeScale]()</h3>
-
-    <h3>[method:AnimationAction setDuration]( [page:Number duration] )</h3>
-
-    <h3>[method:AnimationAction syncWith]( [page:AnimationAction action] )</h3>
-
-    <h3>[method:AnimationAction halt]( [page:Number duration] )</h3>
-
-    <h3>[method:AnimationAction warp]( [page:Number startTimeScale], [page:Number endTimeScale], [page:Number duration] )</h3>
-
-    <h3>[method:AnimationAction stopWarping]()</h3>
-
-    <h3>[method:AnimationMixer getMixer]()</h3>
-
-    <h3>[method:AnimationClip getClip]()</h3>
-
-    <h3>[method:Object3D getRoot]()</h3>
-
-
-    <h2>Static Methods</h2>
+    <h3>[method:AnimationAction setEffectiveWeight]( [page:Number weight] )</h3>
+    <div>
+        Sets the [page:.weight weight] and stops any scheduled fading. This method can be chained.<br /><br />
+ 
+        If [page:.enabled enabled] is true, the effective weight (an internal property) will be set to 
+        this value, too; otherwise the effective weight (directly affecting the animation at this 
```

Comment:
> If [page:.enabled enabled] is true, the effective weight (an internal property) will be set to  this value, too;

Actually rather than replacing the comma (see previous comment) this would read better as 

>If [page:.enabled enabled] is true, the effective weight (an internal property) will also be set to this value;